[2024-08-31T17:45:02.618+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-31T17:45:03.116+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_dag.execution_time manual__2024-08-31T16:50:00.890695+00:00 [queued]>
[2024-08-31T17:45:03.188+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_dag.execution_time manual__2024-08-31T16:50:00.890695+00:00 [queued]>
[2024-08-31T17:45:03.198+0000] {taskinstance.py:2306} INFO - Starting attempt 13 of 14
[2024-08-31T17:45:03.301+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): execution_time> on 2024-08-31 16:50:00.890695+00:00
[2024-08-31T17:45:03.325+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=3193) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-31T17:45:03.332+0000] {standard_task_runner.py:64} INFO - Started process 3195 to run task
[2024-08-31T17:45:03.330+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'simple_dag', 'execution_time', 'manual__2024-08-31T16:50:00.890695+00:00', '--job-id', '343', '--raw', '--subdir', 'DAGS_FOLDER/envioemail.py', '--cfg-path', '/tmp/tmp2bmhw5hh']
[2024-08-31T17:45:03.339+0000] {standard_task_runner.py:91} INFO - Job 343: Subtask execution_time
[2024-08-31T17:45:03.506+0000] {task_command.py:426} INFO - Running <TaskInstance: simple_dag.execution_time manual__2024-08-31T16:50:00.890695+00:00 [running]> on host 19bb8daff211
[2024-08-31T17:45:03.857+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='jorgepons90@gmail.com,facturasjorgepons@gmail.com' AIRFLOW_CTX_DAG_OWNER='Jorge Pons' AIRFLOW_CTX_DAG_ID='simple_dag' AIRFLOW_CTX_TASK_ID='execution_time' AIRFLOW_CTX_EXECUTION_DATE='2024-08-31T16:50:00.890695+00:00' AIRFLOW_CTX_TRY_NUMBER='13' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-31T16:50:00.890695+00:00'
[2024-08-31T17:45:03.861+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-31T17:45:03.915+0000] {logging_mixin.py:188} INFO - 0:00:00.793577
[2024-08-31T17:45:03.961+0000] {base.py:84} INFO - Using connection ID 'my_mssql_conn_id' for task execution.
[2024-08-31T17:45:04.066+0000] {logging_mixin.py:188} INFO - Error: Unsupported parameter type: <class 'datetime.timedelta'>
[2024-08-31T17:45:04.073+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-31T17:45:04.076+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-31T17:45:04.100+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=execution_time, run_id=manual__2024-08-31T16:50:00.890695+00:00, execution_date=20240831T165000, start_date=20240831T174503, end_date=20240831T174504
[2024-08-31T17:45:04.154+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-31T17:45:04.201+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-31T17:45:04.205+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
